src/data/set/finite.lean

```diff
@@ -994,6 +994,33 @@ by { rw ←range_subset_iff at hf, exact (infinite_range_of_injective hi).mono h
 lemma infinite.exists_nat_lt {s : set ℕ} (hs : s.infinite) (n : ℕ) : ∃ m ∈ s, n < m :=
 let ⟨m, hm⟩ := (hs.diff $ set.finite_le_nat n).nonempty in ⟨m, by simpa using hm⟩
 
+lemma arb_large_infinite {S : set ℕ} : S.infinite → ∀ (n : ℕ ), ∃ (m : ℕ), (m ∈ S ∧ m > n) :=
+begin
+  intros h n,
+  apply bex_def.mp,
+  apply set.infinite.exists_nat_lt h,
+end
```

Comment:
This lemma is pretty much identical to `infinite.exists_nat_lt`, so I don't think it's worth having.

Reply:
After removing `arb_large_infinite`, could I then replace the definition of `infinite_iff_arb_large` with the following?

```lean
lemma infinite_iff_arb_large {S : set ℕ }  : S.infinite ↔ ∀ (n : ℕ ), ∃ (m : ℕ), (m ∈ S ∧ n < m) :=
  begin
    split,
    { intro hS,
    have exists_nat_lt := hS.exists_nat_lt,
    finish, },
    { exact infinite_exists_nat_lt, }
  end
  ```